src/parachain/redeem.ts

```diff
@@ -227,16 +227,16 @@ export class DefaultRedeemAPI {
     }
 
     async getFeesToPay(amount: string): Promise<string> {
-        const feePercentage = await this.getFeePercentage();
+        const feePercentage = await this.getFeeRate();
         const feePercentageBN = new Big(feePercentage);
```

Comment:
We can remove this line, since the `getRate()` function returns Big already (the same way you did for issue)

Reply:
Done!